packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/provider.ts

```diff
@@ -260,7 +255,7 @@ export function setupAIProvider() {
     if (options.attachments?.length) {
       promptName = 'debug:action:fal-sd15';
     }
-    return toImage({
+    return textToText({
```

Comment:
In the case of  text or image, image to image, need to call `imagesStream`

Reply:
both types of message have been listened to in https://github.com/toeverything/AFFiNE/pull/6628/files#diff-d2d8b857ef3f0fefca4175360625704b75151205862d68760a5baf53d196c179R48-R49 and mapped in https://github.com/toeverything/AFFiNE/pull/6628/files#diff-e8a9e3cb567d0cc9ab5e33a5ad6cba230be641ad804f7b8440c852fcdbb98333R126-R128